src/generate/renders/CliTopAction.ts

```diff
@@ -65,7 +65,7 @@ export class CliTopAction extends TemplateBase {
                 const keepLineIdx = keepHeaderLines(baseSplit);
                 let hasLoadLogic = false;
                 if (skipLineIdx !== -1) {
-                    for (let i: number = skipLineIdx + 1; i < baseSplit.length; ++i) {
```

Comment:
I remember we had similar issue fixes for this before. why we didn't find out that fix is not working last time? Is it possible to add some scenairo test to make sure it works?

Reply:
though for same issue, they comes from different code's different bug.

The change in below is for testing this bug:
![image](https://user-images.githubusercontent.com/59815250/114358843-f824c300-9ba5-11eb-8263-a9b9bb9a8514.png)